src/json/json-document.ts

```diff
@@ -54,4 +54,8 @@ export class ParsedJsonDocument extends ParsedDocument<Json, Visitor> {
   sourceRangeForNode(_node: Json): SourceRange {
     throw new Error('Not Implemented.');
   }
+
+  stringify() {
+    return JSON.stringify(this.ast, null, 2);
```

Comment:
Yeah, something like that. I didn't look very hard, so there might be a non-SAX parser that does this too. JSON isn't so important in the scheme of things though.